pika/connection.py

```diff
@@ -320,21 +356,6 @@ class ConnectionParameters(Parameters):
     """Connection parameters object that is passed into the connection adapter
     upon construction.
 
-    :param str host: Hostname or IP Address to connect to
```

Comment:
@gmr: I removed the parameters from the `ConnectionParameters` class docstring, because the exact same information was in the constructor's docstring in `ConnectionParameters.__init__`. It was the usual pain point of having to change the same thing in multiple places. What is the benefit of having that same information in both the class and constructor docstrings?